fog/ingest/enclave/impl/src/rng_store.rs

```diff
@@ -24,18 +24,18 @@ type KeySize = U32;
 type ValueSize = U8;
 // BlockSize is a tuning parameter for OMap which must become the ValueSize of
 // the selected ORAM
-type BlockSize = U1024;
+type BlockSize = U2048;
 // This selects an oblivious ram algorithm which can support queries of size
 // BlockSize The ORAMStorageCreator type is a generic parameter to RngStore
-type ObliviousRAMAlgo<OSC> = PathORAM4096Z4Creator<McRng, OSC>;
+type ObliviousRAMAlgo<OSC> = CircuitORAM4096Z2Creator<McRng, OSC>;
 
 // These are the requirements on the storage, this is imposed by the choice of
 // oram algorithm
 
 /// The storage data size which OSC must be able to support
 pub type StorageDataSize = U4096;
 /// The storage meta size which OSC must be able to support
-pub type StorageMetaSize = U64;
+pub type StorageMetaSize = U32;
```

Comment:
this is going down because the bucket size is two now?

Reply:
Yes. Because we're going from 16x4 to 16x2.